lua/lazyvim/plugins/extras/lang/php.lua

```diff
@@ -50,7 +50,7 @@ return {
       local dap = require("dap")
       dap.adapters.php = {
         type = "executable",
-        command = "php-debug-adapter",
+        command = vim.fn.exepath("php-debug-adapter"),
```

Comment:
Is this really needed? I also asked a question in https://github.com/LazyVim/LazyVim/pull/6075. Is that not the case?

Reply:
When I tried it without, there was an error saying it couldn't find the command. Maybe it's because I'm on NixOS and there's something missing from the PATH. Could you try it without the `exepath` and see if it works?

Reply:
Will try that tomorrow because it's pretty late here and turned off pc. On mobile trying to chill a bit at the moment before falling.

Reply:
I tried it without `vim.fn.exepath` and it worked for me without error. Because I don't do any php, I asked AI to generate a simple file. Then did `<leader>db` to toggle breakpoint and `<leader>dc` to start the UI (having enabled the php Extra) and it started correctly and didn't complain about any command missing from PATH. So, maybe this is a NixOS thing?